src/kani-compiler/src/codegen_cprover_gotoc/utils/utils.rs

```diff
@@ -56,8 +56,8 @@ impl<'tcx> GotocCtx<'tcx> {
         loc: Location,
         url: &str,
     ) -> Expr {
-        // We should possibly upgrade this to a warning in the future, but for now emit at least something
-        debug!("codegen_unimplemented: {} at {}", operation_name, loc.short_string());
+        // TODO: This should print a more user friendly warning format.
+        warn!("codegen_unimplemented: {} at {}", operation_name, loc.short_string());
```

Comment:
We'll print warnings for all u supported constructs in every crate we are compiling.

If you want I can try to aggregate the data into one big warning at the end. But it will be a follow up PR